Not working with Big Sur?

I'm not sure whether this is a mistake on my side or a missing feature.

I have Big Sur (11.5.1) and when trying to run some cargo-instruments it fails saying "xctrace" is missing. Indeed, I can't find that binary.

> % xcode-select --install
> xcode-select: error: command line tools are already installed, use "Software Update" to install updates
> % brew install cargo-instruments
> Warning: cargo-instruments 0.4.1 is already installed and up-to-date.
> % cargo instruments -t Allocations
>       Failed instruments errored: xcrun: error: unable to find utility "xctrace", not a developer tool or in PATH
> % ls /Library/Developer/CommandLineTools/usr/bin/xtr*
> zsh: no matches found


Is there something I can do to run cargo-instruments on this machine?

This link solved this problem for me. 

https://apple.stackexchange.com/a/336589

Now I get the following error:
```
    Finished dev [unoptimized + debuginfo] target(s) in 0.06s
   Profiling target/debug/feophant-server with template 'Allocations'
      Failed instruments errored: [ERROR] Run issues detected (trace is still ready to be viewed):
* Failed to gain authorization
```

--- a/README.md
+++ b/README.md
@@ -17,10 +17,23 @@ This crate only works on macOS because it uses [Instruments] for profiling
 and creating the trace file. The benefit is that Instruments provides great
 templates and UI to explore the Profiling Trace.
 
-To install Xcode Instruments, simply install the Command Line Tools:
+To install Instruments, install Xcode from the App Store.  (The Command Line Tools
+you might have already installed while setting up Rust are not sufficient.)
+
+Verify that install by running `xctrace`.  It should print help output.
+
+If you see an error about the active developer directory, check the path
+printed by `xcode-select --print-path`.  It should look as follows:
 
 ```sh
-$ xcode-select --install
+$ xcode-select --print-path
+/Applications/Xcode.app/Contents/Developer
+```
+
+If not, this command will reset the path:
+
+```sh
+$ sudo xcode-select --reset
 ```
 
 ### Compatibility

